Sources/Orbit/Components/Timeline.swift

```diff
@@ -20,12 +20,15 @@ public struct Timeline<Content: View>: View {
                 GeometryReader { geometry in
                     VStack(spacing: 0) {
                         ForEach(preferences.indices.dropFirst(), id: \.self) { index in
-                            preferences[index].style.color
-                                .frame(width: 2)
-                                .frame(
-                                    width: TimelineStepStyle.indicatorDiameter * sizeCategory.ratio,
-                                    height: progressLineHeight(for: index - 1, in: preferences, geometry: geometry)
-                                )
+                            TimelineSegmentLine(
+                                height: progressLineHeight(for: index - 1, in: preferences, geometry: geometry),
+                                startPointStyle: preferences[index - 1].style,
+                                endPointStyle: preferences[index].style
+                            )
+                            .frame(
```

Comment:
The `frame()` could be encapsulated inside the line component.

Reply:
I replaced it with `offset`, not it will be more clear why it is used outside the component